src/Expr/ClosureExpressionVisitor.php

```diff
@@ -101,18 +106,18 @@ public static function getObjectFieldValue(object|array $object, string $field)
      *
      * @return Closure
      */
-    public static function sortByField(string $name, int $orientation = 1, Closure|null $next = null)
+    public static function sortByField(string $name, int $orientation = 1, Closure|null $next = null, bool $orderDateTimeAsScalar = false)
```

Comment:
This is a breaking change for extending classes, isn't it? If yes, let's implement it with `func_get_arg()` instead?

Reply:
Thank you for the suggestion, changed.